.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "sourceType": "unambiguous",
```

Comment:
reading this https://babeljs.io/docs/options#sourcetype wonder whats the reasoning behind choose `unambiguous` vs `module` since it says here `unambiguous can be quite useful in contexts where the type is unknown, but it can lead to false matches because it's perfectly valid to have a module file that does not use import/export statements.` ...